tools/monorepo-utils/src/code-freeze/commands/changelog/lib/index.ts

```diff
@@ -382,21 +382,20 @@ function getTargetBranches(
 		.split( '.' )
 		.map( Number );
 
-	// Check if the target is greater than the trunk version
 	if (
-		targetMajor < currentMajor ||
-		( targetMajor === currentMajor && targetMinor <= currentMinor )
+		targetMajor > currentMajor ||
+		( targetMajor === currentMajor && targetMinor >= currentMinor )
 	) {
 		Logger.notice(
-			`Target version ${ targetVersion } is not greater than trunk version ${ trunkVersion }. Skipping intermediate branches generation.`
+			`Target version ${ targetVersion } is greater than or equal to trunk version ${ trunkVersion }. Skipping intermediate branches generation.`
```

Comment:
I think we could modify this to remove the "generation" portion while we're making changes here.  Branches in this function are just be accumulated in an array and not generated in any way.